arangod/VocBase/LogicalCollection.cpp

```diff
@@ -1381,37 +1393,41 @@ Result LogicalCollection::compareChecksums(VPackSlice checksumSlice, std::string
 
 std::string LogicalCollection::generateGloballyUniqueId() const {
   if (_version < VERSION_33) {
-    return _name; // predictable UUID for legacy collections
+    return name(); // predictable UUID for legacy collections
   }
-  
+
   ServerState::RoleEnum role = ServerState::instance()->getRole();
   std::string result;
   result.reserve(64);
   if (ServerState::isCoordinator(role)) {
-    TRI_ASSERT(_planId != 0);
+    TRI_ASSERT(planId());
     result.append("c");
-    result.append(std::to_string(_planId));
+    result.append(std::to_string(planId()));
   } else if (ServerState::isDBServer(role)) {
-    TRI_ASSERT(_planId != 0);
+    TRI_ASSERT(planId());
     result.append("c");
     // we add the shard name to the collection. If we ever
     // replicate shards, we can identify them cluster-wide
-    result.append(std::to_string(_planId));
+    result.append(std::to_string(planId()));
     result.push_back('/');
-    result.append(_name);
+    result.append(name());
   } else { // single server
     if (isSystem()) { // system collection can't be renamed
-      result.append(_name);
+      result.append(name());
     } else {
-      TRI_ASSERT(_cid != 0);
+      TRI_ASSERT(id());
       result.append("h");
       char buff[sizeof(TRI_server_id_t) * 2 + 1];
       size_t len = TRI_StringUInt64HexInPlace(ServerIdFeature::getId(), buff);
       result.append(buff, len);
       TRI_ASSERT(result.size() > 3);
       result.push_back('/');
-      result.append(std::to_string(_cid));
+      result.append(std::to_string(id()));
     }
   }
   return result;
 }
+
+// -----------------------------------------------------------------------------
+// --SECTION--                                                       END-OF-FILE
+// -----------------------------------------------------------------------------
```

Comment:
Is there a particular reason you are adding these footers everywhere?

Reply:
I find them convenient when scrolling through long files or examining diffs, to know if I missed anything at the end of the file. They do not serve any other purpose.